src/components/controls/choose-metric.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import Radium from "radium";
+import queryString from "query-string";
+import TimeTree from "../framework/svg-time-tree";
+import MutationTree from "../framework/svg-mutation-tree";
+
+@Radium
+class ChooseMetric extends React.Component {
```

Comment:
Possible there's a better name here. Metric is pretty general. Anything that distinguishes this pair of metrics?